book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, person)
+    @rentals.push(Rental.new(date, self, person))
+  end
```

Comment:
- [ ] You are yet to implement this change requested by the previous reviewers. Kindly remove the @rentals.push here to avoid the repetitive addition of the same rental. You can refactor to this as previously suggested:

  ```
  def add_rental(date, person)
    Rental.new(date, self, person)
  end
  ```